<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorCheckForm</class>
 <widget class="QWidget" name="ColorCheckForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="combo_Video">
         <item>
          <property name="text">
           <string>Camera Video0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera Video1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera Video2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>File Video</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/assert/ICON/076.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Open">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Start">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Stop">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_CLose">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_OSDTime">
         <property name="text">
          <string>图像上不叠加时间</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addRect">
         <property name="text">
          <string>添加监控区域</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="comboBox_CheckColor">
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_showCtrl">
         <item>
          <property name="text">
           <string>显示原图</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>显示效果图</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_tthbp">
         <property name="text">
          <string>TTHBP 打开</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_blankCtrl">
         <property name="text">
          <string>黑屏停止检测</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_checkCount">
         <property name="text">
          <string>测试次数:0 黑屏次数:0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>单个文件时长(秒):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_recd_file_time">
         <property name="text">
          <string>3600</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_recd">
         <property name="text">
          <string>开始录像</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>黑屏阀值:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_blackhold">
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>上电延时检测:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_delayCheckDuration">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Camera上电延时:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_delayCameraPowerDuration">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>检测时长:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_checkDuration">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>断电时长:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_powerOffDuration">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Setting">
         <property name="text">
          <string>确定配置</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="imageHLayout" stretch="10,5">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>529</width>
            <height>275</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,4,1,10">
         <item>
          <widget class="QLabel" name="label_ErrTimes">
           <property name="text">
            <string>问题时间点记录:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_ErrTimes"/>
         </item>
         <item>
          <widget class="QLabel" name="label_checkRect">
           <property name="text">
            <string>检测区域:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>261</width>
              <height>149</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="statusBar">
     <property name="text">
      <string>状态信息</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Open</sender>
   <signal>clicked()</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_OpenVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>490</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_CLose</sender>
   <signal>clicked()</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_CloseVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Start</sender>
   <signal>clicked()</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_StartVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Stop</sender>
   <signal>clicked()</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_StopVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_Video</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_combo_Video_Change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_addRect</sender>
   <signal>clicked()</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_AddRect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Setting</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_Setting_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_tthbp</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_TTHBP_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>658</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_blankCtrl</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_BlankCtrl_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_CheckColor</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_combo_CheckColor_Change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>650</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_showCtrl</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_combo_ShowCtrl_Change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_recd</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_recd_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>806</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OSDTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColorCheckForm</receiver>
   <slot>on_button_osdTime_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_OpenVideo_clicked()</slot>
  <slot>on_button_CloseVideo_clicked()</slot>
  <slot>on_button_StartVideo_clicked()</slot>
  <slot>on_button_StopVideo_clicked()</slot>
  <slot>on_combo_Video_Change(QString)</slot>
  <slot>on_button_AddRect_clicked()</slot>
  <slot>on_Label_LinkActivited(QString)</slot>
  <slot>on_button_Setting_clicked(bool)</slot>
  <slot>on_button_TTHBP_clicked(bool)</slot>
  <slot>on_button_BlankCtrl_clicked(bool)</slot>
  <slot>on_combo_CheckColor_Change(QString)</slot>
  <slot>on_combo_ShowCtrl_Change(QString)</slot>
  <slot>on_button_recd_clicked(bool)</slot>
  <slot>on_button_osdTime_clicked(bool)</slot>
 </slots>
</ui>
